package.json

```diff
@@ -23,13 +23,18 @@
     "bookshelf": "^1.2.0",
     "colors": "^1.4.0",
     "cookie-parser": "~1.4.4",
+    "cors": "^2.8.5",
     "debug": "~2.6.9",
     "dotenv": "^8.2.0",
+    "ejs": "^3.1.5",
```

Comment:
Are you using `ejs`? Notice that adding packages you don't need makes other users download and install them when running `npm i` (this is not a good practice) https://www.reddit.com/r/ProgrammerHumor/comments/6s0wov/heaviest_objects_in_the_universe/